flexget/components/estimate_release/estimate_release.py

```diff
@@ -12,32 +34,105 @@ class EstimateRelease:
     for various things (series, movies).
     """
 
-    def estimate(self, entry):
+    def __init__(self):
+        self.task_estimate_config = {}
+
+    @property
+    def schema(self) -> Dict[str, Any]:
```

Comment:
This is not the way to handle sub plugins & interfaces, please look at how discover plugin handles search interface.

Reply:
Sure.  I'll take a look.